community/membership.md

```diff
@@ -1,12 +1,13 @@
 # Community Membership
 
 This document outlines the various responsibilities of contributor roles in
-Argoproj. The Argo project is currently subdivided into four main subprojects:
+Argoproj. The Argo project is currently subdivided into five main subprojects:
```

Comment:
I remember we decided that argo-helm was considered to be an area/scope. See https://github.com/argoproj/argoproj/blob/main/MAINTAINERS.md, all members are scoped.